code/modules/mob/living/carbon/human/human.dm

```diff
@@ -39,6 +39,13 @@
 	GLOB.human_list -= src
 	return ..()
 
+/mob/living/carbon/human/ZImpactDamage(turf/T, levels)
+	if(dna.species.flying_species || (ismoth(src) && dna.features["moth_wings"] != "Burnt Off"))
+		visible_message("<span class='danger'>[src] softly lands on [T].</span>", \
+						"<span class='userdanger'>You softly land on [T].</span>")
+		Knockdown(levels * 50)
+		return

```

Comment:
can you still damage stuff below you by falling on it as a moth (who still has their wings)?